manager/config/integration-tests/datacatalog.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 IBM Corp.
```

Comment:
Why are these in `config/integrations-tests` and not part of the mocks themselves?

Reply:
I think we should keep the integration tests yamls together and not spread over so they don't add confusion. Otherwise the setup for integration tests gets even more complicated having to install 5 yaml files from 5 different directories.